<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simple_Search</class>
 <widget class="QWidget" name="Simple_Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="name_ln">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Введите название</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="name_cb">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentText">
      <string>Точно</string>
     </property>
     <item>
      <property name="text">
       <string>Точно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Совпадение</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="simple_search_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>84</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Искать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
